consensus_pledge_model/logic.py

```diff
@@ -316,9 +316,9 @@ def s_sectors_rewards(params,
 
         # create new dict, and adds the values from shifted reward schedule and
         # newly created reward schedule
-        updated_reward_schedule = {x: new_reward_schedule.get(x, 0.0) + today_reward_schedule.get(x, 0.0)
-                                   for x in set(new_reward_schedule).union(today_reward_schedule)}
-        reward_schedule = updated_reward_schedule
+        reward_days = set(new_reward_schedule | today_reward_schedule)
```

Comment:
Potential bug in here as the objs are dicts rather than set. The union operator on this case requires defining them as sets. Also, make explicit that the set is composed of keys.

`reward_days  = set(new_reward_schedule.keys()) | set(today_rewards_schedule.keys())`